java/flight/flight-core/src/test/java/org/apache/arrow/flight/TestClientMiddleware.java

```diff
@@ -246,12 +246,13 @@ public FlightClientMiddleware onCallStarted(CallInfo info) {
   }
 
   // Used to test that middleware can send and receive multi-valued text and binary headers.
-  static final Map<String, List<byte[]>> EXPECTED_BINARY_HEADERS = new HashMap<String, List<byte[]>>() {{
-      put("x-binary-bin", Arrays.asList(new byte[] {0}, new byte[]{1}));
-    }};
-  static final Map<String, List<String>> EXPECTED_TEXT_HEADERS = new HashMap<String, List<String>>() {{
-      put("x-text", Arrays.asList("foo", "bar"));
-    }};
+  static final Map<String, List<byte[]>> EXPECTED_BINARY_HEADERS = new HashMap<String, List<byte[]>>();
+  static final Map<String, List<String>> EXPECTED_TEXT_HEADERS = new HashMap<String, List<String>>();
+
+  {
```

Comment:
```suggestion
  static {
```

Reply:
Deleted

Reply:
Err, nothing changed here and this is a suggested addition - did you forget to add this?

Reply:
My mistake, just applied suggestion